ocm.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+def call_ocm(endpoint, token, parameters=None):
+    r = requests.get("{}/{}".format("https://api.openshift.com/api/", endpoint),
+            params=parameters, 
+            headers = {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer {}'.format(token),
+                'User-Agent': 'pythonCall'
+                }
+            )
+
+    if r.status_code == 204:
+        return []
+    if r.status_code != 200:
+        raise Exception('''looking up infomation from: {}\n
+                Error Code: {}\n {}'''.format(endpoint, r.status_code, r.url))
+
+    return r.json()
+
+
+def cluster_to_account(token, cluster):
+    requests.packages.urllib3.disable_warnings()
```

Comment:
You can drop this now that you call it at the module level

Reply:
D: I thought I pulled that one! crap.